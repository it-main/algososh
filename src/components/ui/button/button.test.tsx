import { render } from "@testing-library/react";
import { Button } from "./button";
// import renderer from "react-test-renderer";

describe("Тестирование компонента Button", () => {
  it("Компонент с надписью рендерится корректно", () => {
    const { asFragment } = render(<Button text="Развернуть" />);
    expect(asFragment()).toMatchSnapshot();

    // const tree = renderer
    //   .create(<Link page="http://www.facebook.com">Facebook</Link>)
    //   .toJSON();
    // expect(tree).toMatchSnapshot();
  });
});
