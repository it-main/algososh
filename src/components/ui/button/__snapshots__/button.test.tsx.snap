// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Тестирование компонента Button Компонент с надписью рендерится корректно 1`] = `
<DocumentFragment>
  <button
    class="text text_type_button text_color_primary button undefined false "
    type="button"
  >
    <p
      class="text undefined"
    >
      Развернуть
    </p>
  </button>
</DocumentFragment>
`;
